src/Main.elm

```diff
@@ -0,0 +1,91 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
```

Comment:
Don't expose `Msg`'s constructors (remove the `(..)`) and `main` - they shouldn't be used externally.